hub/values.yaml

```diff
@@ -323,6 +325,8 @@ jupyterhub:
             return labels
 
           async def start(self):
+            # TODO: REMOVE BEFORE MERGING 3963 TO PROD!!!!!!!!!
+            sleep(45)
```

Comment:
You want `await asyncio.sleep(45)` here - otherwise the *entire process* will be blocked, and the code in the progress handler will not get a chance to execute until after this sleep call returns.

Reply:
ah, i was curious about that...  and it makes sense.  :)